netket/_ss.py

```diff
@@ -0,0 +1,411 @@
+import sys
+
+import numpy as _np
+
+import netket as _nk
+from netket._core import deprecated
+from netket.operator import local_values as _local_values
+from netket.operator import local_values_op_op as _local_values_op_op
+from netket.operator import der_local_values as _der_local_values
+from netket.stats import (
+    statistics as _statistics,
+    covariance_sv as _covariance_sv,
+    subtract_mean as _subtract_mean,
+)
+
+
+def info(obj, depth=None):
+    if hasattr(obj, "info"):
+        return obj.info(depth)
+    else:
+        return str(obj)
+
+
+def make_optimizer_fn(arg, ma):
```

Comment:
we should move this to the optimizer module and also remove it from _vmc...

Reply:
I think the best option would be to have a submodule with shared functions (and possibly a base class) for the driver classes. The `make_..._fn` are specifically written for the needs of the driver.

Reply:
Also, can we give `_ss.py` a more descriptive name? Abbreviating `_vmc.py` is fine, since VMC is usually used as an acronym. I don't think that's treu for *steady state*. Why not call it `_steadystate.py`.

Reply:
> I think the best option would be to have a submodule with shared functions (and possibly a base class) for the driver classes. The `make_..._fn` are specifically written for the needs of the driver.

I agree, but maybe it's best to do this in another PR. Actually I think we should just remove make_... and instead have converrters from (say) optimizer x to netket optimizer